chicagoland.pandemicresponsecommons.org/portal/gitops.json

```
@@ -475,6 +510,118 @@
             {
               "field": "reporting_org",
               "name": "Reporting Organization"
+            },
+            {
+              "field": "state_region_description",
+              "name": "Region"
```

```suggestion
              "name": "Hospital Region"
```
this just got changed in QA

---

ah i missed that! good catch